lib/ReactViews/FeatureInfo/FeatureInfoSection.tsx

```diff
@@ -257,6 +259,14 @@ export class FeatureInfoSection extends React.Component<FeatureInfoProps> {
       );
     }
 
+    // Add activeStyle property
+    if (TableMixin.isMixedInto(this.props.catalogItem)) {
+      const activeStyleId = this.props.catalogItem.activeStyle;
+      terria.activeStyle = this.props.catalogItem.styles.find(
+        (style) => style.id === activeStyleId
+      );
```

Comment:
@mwu2018 the danger in exposing the entire style object is that the catalog maintainer can now depend on any member of the style object. If we decide to change any of it in the future, it'll silently break the template. So ideally, we should restrict the members we export. I recommend you export just the `id` for now.

Reply:
Done. Please take another look.